app/(main)/league/[leagueId]/entry/all/page.tsx

```diff
@@ -194,20 +193,8 @@ const Entry = ({
                     />
                   </section>
                 );
-              })}
+              })}              
```

Comment:
Can you reverse this white space change please?

Reply:
@choir27 there should be no space or double the space?

Reply:
@choir27 removed white space. let me know if that's not what you wanted.